regexes/oss.yml

```diff
@@ -425,6 +425,9 @@
   name: 'iOS'
   version: ''
 
+- regex: 'RSSRadio/'
+  name: 'iOS'
+  version: '$1'
```

Comment:
This one could be merged with the one 2 above: `'Podcasts/(?:[\d\.]+)|Instacast(?:HD)?/(?:\d\.[\d\.abc]+)|Pocket Casts, iOS|Overcast|Castro|Podcat|i[cC]atcher'`. Also the `$1` would be useless here, as the regex can't match anything

Reply:
Good point. I'll try it. Should I rebase and ammend my commits?

Reply:
no need to rebase or amend. will be squashed when merging